.buildkite/scripts/common.sh

```diff
@@ -69,3 +71,32 @@ with_Terraform() {
     chmod +x ${WORKSPACE}/terraform
     terraform version
 }
+
+google_cloud_auth() {
+    secretFileLocation=$(mktemp -d -p . -t "${TMP_FOLDER_TEMPLATE_BASE}.XXXXXXXXX")/google-cloud-credentials.json
+    echo "${PRIVATE_CI_GCS_CREDENTIALS_SECRET}" > ${secretFileLocation}
+    gcloud auth activate-service-account --key-file ${secretFileLocation} 2> /dev/null
+    export GOOGLE_APPLICATIONS_CREDENTIALS=${secretFileLocation}
+}
+
+upload_packages_to_gcp_backet() {
+    pattern=${1}
+    baseUrl="gs://${JOB_GCS_BUCKET}/${REPO}"
+    bucketUrlCommit="${baseUrl}"/commits/${BUILDKITE_COMMIT}
+    bucketUrlDefault="${baseUrl}"/snapshots
+
+    if [[ ${BUILDKITE_PULL_REQUEST} != "false" ]]; then
+        bucketUrlDefault="${baseUrl}"/pull-requests/pr-${BUILDKITE_COMMIT}
+    fi
+
+    for backetUrl in "${bucketUrlCommit}" "${bucketUrlDefault}"; do
+        gsutil -m -q cp -a public-read -r ${pattern} "${backetUrl}"
+    done
+}
+
+cleanup() {
+    echo "Deleting temporal files..."
+    cd ${WORKSPACE}
```

Comment:
Depending on where the credentials is stored, this `cd` should be changed.

WORKSPACE var points to the bin folder, but probably this is stored at the buildkite workspace folder (repository folder).

Reply:
fixed above, nothing to fix here